build-tools-internal/settings.gradle

```diff
@@ -0,0 +1,7 @@
+dependencyResolutionManagement {
+    versionCatalogs {
+        create("buildLibs") {
```

Comment:
Couldn't this just be `buildLibs { }`? Any benefit from explicitly calling `create()` here?

Reply:
true. fixed. looked at the Kotlin DSL example :)